blocks/init/src/blocks/wrapper/wrapper.js

```diff
@@ -10,9 +10,6 @@ export const Wrapper = (props) => {
   const {
     props: {
       attributes,
-      attributes: {
-        hasWrapper,
-      },
     },
     children,
   } = props;
```

Comment:
Can these props be  destructured in function arguments?

Reply:
not in a pretty an useful way